packages/eslint-plugin-svelte/src/types.ts

```diff
@@ -108,6 +109,18 @@ export interface PartialRuleMetaData {
 	deprecated?: boolean;
 	replacedBy?: string[] | { note: string };
 	type: 'problem' | 'suggestion' | 'layout';
+	/**
+	 * Conditions to determine whether this rule should be applied.
+	 * Multiple conditions can be specified as array, and the rule will be applied if any one of them matches (logical OR).
+	 * If not specified, the rule will be applied to all files.
+	 */
+	conditions?: {
+		svelteVersion?: string; // e.g.: '>= 5.0.0'
+		fileTypes?: SvelteContext['fileType'][];
+		runes?: SvelteContext['runes'];
```

Comment:
Why not just `boolean`? :D

Reply:
This is because making related parts automatically produce type errors when one part is modified makes refactoring easier.

Reply:
I don't know, seems to be too clever when it's just a boolean :D But I don't think this is too important :)